### PR TITLE
Update apb command to latest status in README

### PR DESCRIPTION
**Pull request by mdshuai**
apb command have changed, need update README.

some parameter also not support, like:
```
(apb) [root@dhcp-140-98 dma]# apb init my_apb -p name=my_apb_param,type=string,default=foo_param,description="sample description" --async=optional --bindable --organization my_organization
usage: apb [-h] [--debug] [--project BASE_PATH]
           {init,help,prepare,push,bootstrap,list,remove,build} ...
apb: error: unrecognized arguments: --organization my_organization
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,18 +98,19 @@ subcommand:
...
 This gives us the following example apb.yaml 
 ```
+id: c170da42-cb2b-4285-acab-260c1d788fd0
 name: my_apb
 image: my_organization/my_apb
 description: This is a sample application generated by apb init
 bindable: True
 async: optional
-parameters:
+parameters:
```
Comment (leifmadsen) [inline README.md:112]: Trailing whitespace? (sorry, not trying to nitpick too much :))

Comment (rthallisey) [reply README.md:112]: This should be ```parameters: []```. The README in apb-examples is more up to date. Thanks for the patch 👍

**Review by rthallisey — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mdshuai) [reply README.md:112]: @leifmadsen @rthallisey you both are right. But what I paste here is auto generated by `apb init` command, I don't change anything. I also find the file is invalid yaml. so this must be bug for apb init. you can try it.
```
(apb) [root@dhcp-140-98 dma]# apb init my_apb -p name=my_apb_param,type=string,default=foo_param,description="sample description" --async=optional --bindable --org my_organization
Initializing /tmp/dma/my_apb for an APB.
Generating playbook files
Successfully initialized project directory at: /tmp/dma/my_apb
Please run *apb prepare* inside of this directory after editing files.
(apb) [root@dhcp-140-98 dma]# cat my_apb/apb.yml 
id: b30a594e-94a8-4c6b-b4ca-29b9d465db55
name: my_apb
image: my_organization/my_apb
description: This is a sample application generated by apb init
bindable: True
async: optional
parameters: 
- name: my_apb_param
    description: sample description
    type: string
    default: foo_param
```

Comment (rthallisey) [reply README.md:112]: @mdshuai I misread the indent on line 113. If it's generating an invalid yaml can you open and issue for it?

Comment (mdshuai) [reply README.md:112]: @rthallisey Create issue, https://github.com/fusor/ansible-playbook-bundle/issues/65

**Review by jmrodri — APPROVED** [review]
Looks good to me

**Pull request merged**